_posts/en/newsletters/2022-10-26-newsletter.md

```diff
@@ -0,0 +1,270 @@
+  with a [post][sanders ephemeral] to the Bitcoin-Dev mailing containing
+  a proposal for a new type of [anchor output][topic anchor outputs].  A
+  v3 transaction could pay zero fees but contain an output paying the
+  script `OP_TRUE`, allowing anyone to spend it under the consensus
+  rules in a child transaction.  The zero-fee parent transaction would
+  only be accepted if it was part of a transaction package which also
+  contained the child transaction spending the OP_TRUE output.
+
+    Described advantages of this proposal is that it eliminates the need
+    to use one-block relative timelocks (called `1 OP_CSV` after the
+    code used to enable them) to prevent [transaction pinning][topic
+    transaction pinning] and allows anyone to fee bump the parent
+    transaction (similar to an earlier [fee sponsorship][topic fee
+    sponsorship] proposal).
+
+    Jeremy Rubin [replied][rubin ephemeral] in support of the proposal
+    but noted that it doesn't work for any contract that can't use v3
+    transactions.  Several other developers also discussed the concept,
+    all of them seeming to find it appealing as of this writing.
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 24.0 RC2][] is a release candidate for the
+  next version of the network's most widely used full node
+  implementation.  A [guide to testing][bcc testing] is available.
+
+  **Warning:** this release candidate includes the `mempoolfullrbf`
+  configuration option which several protocol and application developers
+  believe could lead to problems for merchant services as described
+  in the newsletters for this week and last week.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #23443][] p2p: Erlay support signaling FIXME:harding
```

Comment:
Note: I suggest including a clarification (just a few words maybe) that transaction reconciliation functionality has *not* been added yet, only signaling/negotiation. Commenting because people seemed to be really confused by this on Twitter.

Users should *not* pull from master and turn this on thinking they're enabling Erlay on their node. If someone starts a node with this on right now and it's still running when Bitcoin Core releases software with Erlay fully enabled, their node might get disconnected because they're signaling support for Erlay but not actually doing things. (This goes for any other Erlay PRs merged before completion).